### DISCUSSION TITLE
Error: webgl is not supported by the browser is raised even though webgl is supported.

### DISCUSSION THREAD

**Discussion by a-c-sreedhar-reddy** [pixijs/pixijs#8112] [Q&A]
Hi,

Some times Pixi throws the above error saying WebGL is not supported. 

But a pixi application was created previously on the same machine. What could the possible reasons for this?

This happened on Chrome 97.0.4692 with PIXI version: v6.1.1

**Comment by domis86** [comment]
Please provide more info:

* machine specifications
* browser info
* runnable example (pixiplayground, jsfiddle, codesandbox etc) which you use to reproduce the error.

**Comment by themoonrat** [comment]
A screenshot of visiting https://webglreport.com/ on the device would be very useful.

Sometimes browsers update themselves to disable certain features due to buggy GPU drivers and such. It'll be interesting to see what shows up on the above website.
